package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "codesupport-api",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
```

Comment:
Can this be `MIT` please, sorry should have specified in the ticket.

Reply:
Updated it now.